app/src/main/java/org/oppia/app/topic/practice/practiceitemviewmodel/TopicPracticeSkillSummaryViewModel.kt

```diff
@@ -0,0 +1,9 @@
+package org.oppia.app.topic.practice.practiceitemviewmodel
+
+import org.oppia.app.model.SkillSummary
+import org.oppia.app.topic.practice.TopicPracticeFragment
+
+/** Skill summary view model for the recycler view in [TopicPracticeFragment]. */
+class TopicPracticeSkillSummaryViewModel(
+  val skillSummary: SkillSummary
+) : TopicPracticeItemViewModel()
```

Comment:
Write this in single line:

`class TopicPracticeSkillSummaryViewModel(val skillSummary: SkillSummary) : TopicPracticeItemViewModel()`

Reply:
Done.